akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -22,6 +25,85 @@ object ORMap {
    */
   def unapply[A, B <: ReplicatedData](m: ORMap[A, B]): Option[Map[A, B]] = Some(m.entries)
 
+  sealed trait DeltaOp extends ReplicatedDelta with RequiresCausalDeliveryOfDeltas {
+    type T = DeltaOp
+    override def zero: DeltaReplicatedData
+  }
+
+  private[akka] final val VanillaORMapTag = 0
+  private[akka] final val PNCounterMapTag = 1
+  private[akka] final val ORMultiMapTag = 2
+  private[akka] final val ORMultiMapDeltaDeltaTag = 3
+  private[akka] final val LWWMapTag = 4
+
+  /**
+   * INTERNAL API
+   */
+  @InternalApi private[akka] def tagDelta(op: DeltaOp)(tag: Int) = op match {
```

Comment:
Visual aesthetics ;) Wanted to have something like `map { tagDelta(deltaTag) _ }. Forgot that Scala does not do autocurrying like Haskell or ML. No need to fix, as deltas are now tagged at the source and this function is no more...